plugins/dynamic-plugins-info/src/components/DynamicPluginsTable/DynamicPluginsTable.test.tsx

```diff
@@ -45,108 +62,22 @@ describe('DynamicPluginsTable', () => {
     const internalCells = Array.from(
       container.querySelectorAll('tbody tr > td:nth-child(4)'),
     );
-    expect(nameCells.length).toBe(5);
-    expect(nameCells[0].textContent).toBe(
-      '@janus-idp/backstage-plugin-aap-backend-dynamic',
-    );
-    expect(nameCells[2].textContent).toBe(
-      '@janus-idp/backstage-plugin-bulk-import',
-    );
-    expect(versionCells[0].textContent).toBe('');
-    expect(versionCells[2].textContent).toBe('');
-    expect(enabledCells[0].textContent).toBe('No');
-    expect(enabledCells[2].textContent).toBe('No');
-    expect(internalCells[0].textContent).toBe('Yes');
-    expect(internalCells[2].textContent).toBe('Yes');
-  });
-
-  it('supports filtering by a simple text search', async () => {
```

Comment:
Removed these tests, as we can assume `@backstage/core-components` adequately tests `Table`.